src/BridgedRoom.ts

```diff
@@ -676,12 +676,36 @@ export class BridgedRoom {
             return;
         }
 
+        let sendAsLink = false;
+        const authToken = this.SlackClient?.token;
+
+        if (!authToken) {
+            log.error("We have no client (or token) that can handle this file, sending as link");
+            sendAsLink = true;
+        } else if (maxUploadSize && file.size > maxUploadSize) {
+            log.warn(`File size too large (${file.size / 1024}KB > ${maxUploadSize / 1024} KB)`);
```

Comment:
KiB when dividing by 1024.
```suggestion
            log.warn(`File size too large (${file.size / 1024}KiB > ${maxUploadSize / 1024} KiB)`);
```

Reply:
:+1: i get those mixed up